### ISSUE TITLE
Content blocks cannot be used more than once

### ISSUE THREAD

**Issue by brentonhouse** [barc/express-hbs#78]
If I am defining a block like this:  

``` handlebars
{{#contentFor "page_title"}}
    Dashboard Home
{{/contentFor}}
```

And using it like this:

``` html
<title>{{{block "page_title"}}}</title>
<h1>{{{block "page_title"}}}</h1>
```

The second usage doesn't work.   It seems after the first time it is used, it is cleared out (to free memory).

https://github.com/barc/express-hbs/blob/master/lib/hbs.js#L37

``` javascript
this.blocks[name] = null;
```

**Comment by bradvogel** [comment]
+1. Here's another common use case for content blocks that breaks because of this bug:

```
    {{#if (block 'pageTitle')}}
    <title>{{block 'pageTitle'}}</title>
    {{/if}}
```

**Comment by bradvogel** [comment]
ping on this :)

**Comment by jtwebman** [comment]
Is this still an issue with the latest. I remember seeing some cache stuff in there.

**Comment by ErisDS** [comment]
Closing out all old issues. If this is still an issue with the latest version please let us know.

**Comment by skeggse** [comment]
This does appear to be fixed, at least on `2.3.0` (didn't test others).
